src/main/java/com/openlattice/client/RetrofitFactory.java

```diff
@@ -134,16 +122,23 @@ public static final Retrofit newClient( Retrofit.Builder retrofitBuilder ) {
                 .addCallAdapterFactory( callFactory );
     }
 
-    public static final OkHttpClient.Builder okhttpClientWithLoomAuth( Supplier<String> jwtToken ) {
+    public static OkHttpClient.Builder okHttpClient() {
         return new OkHttpClient.Builder()
-                .addInterceptor( chain -> chain
-                        .proceed( chain.request().newBuilder().addHeader( "Authorization", "Bearer " + jwtToken.get() )
-                                .build() ) )
                 .readTimeout( 0, TimeUnit.MILLISECONDS )
                 .writeTimeout( 0, TimeUnit.MILLISECONDS )
                 .connectTimeout( 0, TimeUnit.MILLISECONDS );
     }
 
+    public static final OkHttpClient.Builder okhttpClientWithLoomAuth( Supplier<String> jwtToken ) {
```

Comment:
can we rename this to like `okHttpClientWithOpenLatticeAuth` or something? (and same for `decorateWithOpenLatticeFactories`)

Reply:
Oh, right, didn't even noticed the naming here.